server/api/users.js

```diff
@@ -20,6 +20,12 @@ router.put('/:id/profile', async (req, res, next) => {
       include: [{model: Event, include: {model: EventType}}]
     })
     if (user) {
+      await user.setSkills(
+        req.body.skills.reduce(
+          (skillsArr, currSkill) => [...skillsArr, currSkill.id],
```

Comment:
I'm not following why we're stripping just the id's out of here? don't we want to setSkills on the entire skill object so we can access all of the props?